bundle/manifests/rhdh.redhat.com_orchestrators.yaml

```diff
@@ -309,7 +328,7 @@ spec:
                       to use to deploy the operator.
                     properties:
                       channel:
-                        default: fast-1.3
+                        default: fast-1.2
```

Comment:
Seems file config/crd/bases/rhdh.redhat.com_orchestrators.yaml is outdated. Should be fast-1.3 here?

Reply:
Will update. Thanks for pointing it out.